### PR TITLE
Rhinofi update: Blocklist implemented

### PR DESCRIPTION
**Pull request by sekuba**
Resolves L2B-4634

### CODE REVIEW

**Review by lucadonnoh — CHANGES_REQUESTED** [review]
add in the withdrawal section this info and a risk warning

File: packages/config/src/common/exits.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { formatSeconds } from '@l2beat/shared-pure'
 
 import { ScalingProjectRisk } from './ScalingProjectRisk'
 import { ScalingProjectTechnologyChoice } from './ScalingProjectTechnologyChoice'
+import { text } from 'stream/consumers'
```
Comment (lucadonnoh) [inline packages/config/src/common/exits.ts]: ?

**Review by lucadonnoh — APPROVED** [review]

**Review by lucadonnoh — CHANGES_REQUESTED** [review]
image
let's also add a phrase here that users can be censored explicitely

**Comment by sekuba** [comment]
oke

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**